mrobosub_localization/src/localization.py

```diff
@@ -90,20 +96,25 @@ def imu_callback(self, msg):
         pitch = degrees(-euler[1]) - self.pitch_offset
         roll = degrees(euler[0]) - self.roll_offset
 
-        self.yaw_pub.publish(yaw)
-        self.pitch_pub.publish(pitch)
-        self.roll_pub.publish(roll)
-
-    def iekf_callback(self, msg: Float64):
-        roll = msg.data['roll']
-        pitch = msg.data['pitch']
-        yaw = msg.data['yaw']
-        velocity = msg.data['velocity']
-        position = msg.data['position']
-
-        self.roll_pub.publish(roll)
-        self.pitch_pub.publish(pitch)
-        self.yaw_pub.publish(yaw)
+        self.iekf_class.add_imu_measurement() #input should be measured acceleration and gyro (Imu.linear_acceleration ?)
```

Comment:
We'll get you what to expect in terms of the imu message soon, but don't merge till then